### PR TITLE
Flush conntrack entries

### PR DESCRIPTION
**Pull request by fcrisciani**
Properly flush conntrack entries when the container connectivity is removed
This will avoid that previous flows of other containers will affect new ones

Related to docker/docker/issues/8795

Test will come on docker/docker repo

### CODE REVIEW

File: iptables/conntrack.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+)
+
+const (
+	conntrackNetlinkModule = "nf_conntrack_netlink"
+)
+
+// IsConntrackProgrammable returns true if the handle supports the NETLINK_NETFILTER and the base modules are loaded
+func IsConntrackProgrammable(nlh *netlink.Handle) bool {
+	conntrackInitCheck()
+	return nlh.SupportsNetlinkFamily(syscall.NETLINK_NETFILTER) && isConntrackAvailable
```
Comment (aboch) [inline iptables/conntrack.go]: Given you attempt to create the netlink handle with the syscall.NETLINK_NETFILTER regardless:
```
+	fams := []int{syscall.NETLINK_ROUTE, syscall.NETLINK_NETFILTER}
```
this check here is redundant `nlh.SupportsNetlinkFamily(syscall.NETLINK_NETFILTER)` .

Comment (fcrisciani) [inline iptables/conntrack.go]: that is what happen in the bridge right? Is that possible that in the future someone calls this method with an handle created manually and does not set the proper flag?

Comment (aboch) [inline iptables/conntrack.go]: That is true. This function cannot assume it will be passed the default handle.  :+1:

Comment (aboch) [inline iptables/conntrack.go]: My next comment was to instead do a runtime check in `ns/init_linux.go` before including the `syscall.NETLINK_NETFILTER` in the family list.

**Comment by aboch** [comment]
LGTM

**Pull request merged**